package.json

```diff
@@ -27,5 +28,8 @@
     "gulp": "^3.9.1",
     "gulp-uglify": "^1.5.4",
     "pump": "^1.0.1"
+  },
+  "resolutions": {
+    "graceful-fs": "^4.2.4"
```

Comment:
@NimperX - apologies for not responding to this PR sooner, I haven't looked at this project in a long time. Would you be able clarify what the purpose of adding the graceful-fs package was, along with adding `npx npm-force-resolutions`?